.changeset/tasty-moles-wonder.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs': minor
+---
+
+Add a new config for `allowedIframeHosts` to allow settings custom allowed attributes
```

Comment:
Sorry for the nitpick,  but it might be useful to include an example of the updated config in the changelog to make it easier for folks to understand the change, rather than jumping into the docs. What you have in the how-to could be replicated here.